egs/swbd/s5c/local/nnet3/prepare_multistyle_data.sh

```diff
@@ -0,0 +1,242 @@
+#!/bin/bash
+
```

Comment:
put the normal apache header on this please, and your name.

Reply:
Also can you please add an arg `use_ivectors=true` that wil give the option to skip all the i-vector-related parts of the script?  We may want to use this option later on when we switch to online CMN, but for now I guess we can leave that issue separate, I suppose there is no need to mix it up with this.